src/status_im/contexts/profile/edit/modal/view.cljs

```diff
@@ -0,0 +1,266 @@
+(ns status-im.contexts.profile.edit.modal.view
```

Comment:
This namespace has been recovered, this is the "Edit profile" screen. Previously we used this for the onboarding flow when the user added their details.

The implementation has been modified since we need to validate different things now. Previously the user needed to fill all fields, now, the user can update their profile by just __changing__ one field (not filling, because they are already filled). The fields are: profile photo, color selected and name displayed.

Reply:
There are some things I didn't want to change due to the already existing complexity to comply withthe new requirements, such as ratoms and global ratoms defined for the floating button

Reply:
BTW, Reagent is being used in a good way, so re-renders are minimized compared to the original version